src/instances.ts

```diff
@@ -459,7 +472,14 @@ export function getCustomTransformers(
     getCustomTransformers = customerTransformers;
   }
 
-  return getCustomTransformers(program, getProgram);
+  let transformers = getCustomTransformers?.(program, getProgram);
+  if (loaderOptions.transpileOnly) {
+    (transformers ??= {}).before = [
+      getSetImpliedNodeFormatTransformer(instance, loaderContext, getProgram),
+      ...(transformers?.before ?? []),
```

Comment:
this reminds me, I wonder whether we should bump the `target` for the emitted JavaScript of ts-loader at some point.   I suspect it's quite an old version